docs/pages/connect-your-client/tsh.mdx

```diff
@@ -878,5 +880,13 @@ exit
 end of session playback
 ```
 
+## Uninstalling tsh
+
+If you installed on MacOS with `brew` run `brew uninstall teleport`.  
```

Comment:
I'm not sure we want to be highlighting the presence of the unsupported Homebrew package.